.github/workflows/sonarcloud.yaml

```diff
@@ -7,13 +7,10 @@ on:
   pull_request:
```

Comment:
Can narrow to relevant types of pull requests

```
  pull_request:
    types: [ opened, synchronize, reopened ]
```

Reply:
Done.